### ISSUE TITLE
"first broker login" missing for keycloak_authentication_bindings resource

### ISSUE THREAD

**Issue by eXonius** [keycloak/terraform-provider-keycloak#995]
It seems like all authentication user flows can be bound to with the keycloak_authentication_bindings resource except for the one called "first broker login". 

Is the "first_broker_login_flow" option missing as argument or how am I supposed to bind a newly created flow to to it? I can do it in the GUI by clicking on a new flow and then "Action"->"Bind flow".

<img width="1098" alt="image" src="https://github.com/user-attachments/assets/33e8ce15-e4fc-4c22-a57f-51d23c245e13">

**Comment by jermarchand** [comment]
Hi,

Do you try to use : 
* [first_broker_login_flow_alias for keycloak_oidc_identity_provider](https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/oidc_identity_provider#first_broker_login_flow_alias)
* [first_broker_login_flow_alias for keycloak_saml_identity_provider](https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs/resources/saml_identity_provider#first_broker_login_flow_alias)

br,

**Comment by eXonius** [comment]
Yes using those on each identity provider is the workaround I'm using since it's not possible to set the standard flow.

**Comment by sschu** [comment]
Implemented in https://github.com/keycloak/terraform-provider-keycloak/pull/1084
